cultcargo/images/ddfacet/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN apt-get update && \
 
 {pre_install}
 
-RUN python{default_python_version} -mpip install --use-pep517 --no-cache-dir -U matplotlib dask-ms python-casacore casadata ninja
+RUN python{default_python_version} -mpip install --use-pep517 --no-cache-dir -U matplotlib mpl-tools dask-ms python-casacore casadata ninja
```

Comment:
is the `-mpip` a typo?

Reply:
No, that's the official way to invoke pip via python.